### PR TITLE
Feature/admin module

### PR DESCRIPTION
**Pull request by NurmukhammadMakhmudov**
https://user-images.githubusercontent.com/80941197/125972639-495d4d87-a45c-403e-b175-2752e49188a9.mov

### CODE REVIEW

**Comment by NurmukhammadMakhmudov** [comment]
now pulled update project version and changed the colours

Uploading ekrana-2021-07-16-v-212605_d4bmbnbk.mp4…

**Comment by RomPavel** [comment]
Please change target branch for this PR from **master** to **dev**

**Comment by RomPavel** [comment]
Lets make first row (row with column titles) fixed, so it will not hide on scroll

**Comment by RomPavel** [comment]
We use white theme now, please change

File: exadel-trainee-app/src/mocks/mock_adminProfile.ts [hunk 1]
```diff
@@ -0,0 +1,752 @@
+export interface PeriodicElement {
```
Comment (RomPavel) [inline exadel-trainee-app/src/mocks/mock_adminProfile.ts:1]: Lets have common file name style
admin-profile.mock.ts
like user.utils.mock.ts

File: exadel-trainee-app/src/app/models/admin-profile/admin-profile.component.ts [hunk 2]
```diff
@@ -1,10 +1,44 @@
...
+  ],
 })
 export class AdminProfileComponent implements OnInit {
+  notAssign = false;
+
+  constructor(public dialog: MatDialog) {}
+
+  displayedColumns: string[] = ['Position', 'Level', 'Date', 'Coach'];
+
+  displayedColumns1: string[] = ['Position', 'Level', 'Date', 'button'];
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/models/admin-profile/admin-profile.component.ts:26]: Why you need duplicate columns

File: exadel-trainee-app/src/app/models/admin-profile/admin-dialog/admin-dialog.component.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  level: string;
+}
+
+@Component({
+  selector: 'app-admin-dialog',
+  templateUrl: './admin-dialog.component.html',
+  styleUrls: ['./admin-dialog.component.scss'],
+})
+export class AdminDialogComponent implements OnInit {
+  coachList: CoachList[] = [
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/models/admin-profile/admin-dialog/admin-dialog.component.ts:15]: Lets move it also to mocks file

**Review by RomPavel — APPROVED** [review]

**Pull request merged**